src/status_im2/contexts/shell/cards/style.cljs

```diff
@@ -15,6 +15,7 @@
   {:width            160
    :height           160
    :border-radius    16
+   :overflow         :hidden
```

Comment:
why this is needed?

Reply:
It's necessary to make the banner image's border-radius.